core/ir/aarch64/instr.c

```diff
@@ -1,5 +1,5 @@
 /* **********************************************************
```

Comment:
The PR title and description made me think you were unmarking eret as a branch altogether.  Worth clarifying it's still marked as an indirect branch; maybe worth changing the title to "Mark erat as indirect branch instead of return".

Reply:
I believe the title is concise, since it mentions only the part about unmarking as return which is true, so a reader shouldn't assume anything about its treatment as an indirect branch. But it's a fair question, and the PR description should answer it, so clarified in the PR description.